ASCIIToSVG.php

```diff
@@ -1480,6 +1516,7 @@ private function parseBoxes() {
 
     /* Anything after this is not a subgroup */
     $this->svgObjects->popGroup();
+    $this->svgObjects->rearrangeBoxes();
```

Comment:
Please remove this line as the logic performed by this call should instead just be performed by `SVGGroup::render`.